metadata/src/main/java/org/apache/kafka/metadata/FinalizedControllerFeatures.java

```diff
@@ -66,11 +66,9 @@ public boolean equals(Object o) {
 
     @Override
     public String toString() {
-        StringBuilder bld = new StringBuilder();
-        bld.append("{");
-        bld.append("featureMap=").append(featureMap.toString());
-        bld.append(", epoch=").append(epoch);
-        bld.append("}");
-        return bld.toString();
+        return "{" +
```

Comment:
If we bother updating this method, let's make it consistent with the other ones.

Reply:
Hey @mimaison , I don't exactly get the suggestion here. Could you pls elaborate the same?
Thanks

Reply:
In most other classes in this module `toString()` return a value with this format `<CLASS_NAME>(<FIELDS>)` while this `toString()` method returns `{<FIELDS>}`.

Since you're touching this method, let's make them consistent.

Reply:
Thanks for pointing that out. I updated the same.